.github/workflows/ci_tests.yml

```diff
@@ -53,6 +53,36 @@ jobs:
     - name: Run tests (App) [TypeScript]
       run: yarn test  --ci app
 
+  test-digital:
```

Comment:
While this'll be faster in terms of the total-test-time, it will clog up the Checks in each PR, we should probably just separate frontend vs non-frontend tests, so `test-app` and `test-frontend` only